index.html

```diff
@@ -9,6 +9,17 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
+    <div class="mobile-menu">
+      <div class="mobile-menu-close">
+        <i class="mobile-menu-close-btn fas fa-times"></i>
+      </div>
```

Comment:
Kindly include `mobile menu` `<div>` tag inside `<nav>` tag as the last reviewer mentioned.